Backend/src/main/java/com/project/omega/controller/UserController.java

```diff
@@ -68,7 +68,7 @@ public ResponseEntity updateById(@PathVariable(value = "id") Long id,
             userToUpdate.setEmail(userInDb.getEmail());
         }
 
-        if (!updatedUser.getPassword().isEmpty()) {
+        if (updatedUser.getPassword() != null) {
```

Comment:
Shouldn't we also check that it's not an empty string?

Reply:
that is true i forgot Java wont check with null. nice catch